src/main/java/rx/AsyncEmitter.java

```diff
@@ -29,8 +29,10 @@
  *
  * @param <T> the value type to emit
  * @since (if this graduates from Experimental/Beta to supported, replace this parenthetical with the release number)
+ * @deprecated as of 1.2.1 because Async prefix of this class potentially misleading. Use {@link Emitter} instead.
  */
 @Experimental
+@Deprecated
 public interface AsyncEmitter<T> extends Observer<T> {
```

Comment:
You could make this extend from `Emitter` to avoid having two implementations of the operator.